config/default_records/plate_purposes/014_bge.wip.yml

```diff
@@ -0,0 +1,21 @@
+# Plate purposes for BGE
+---
+BGE Cherrypick:
+  type: PlatePurpose::Input
+  target_type: Plate
+  default_state: passed
+  stock_plate: false
+  cherrypickable_target: true
+  size: 96
+BGE Lib XP2:
+  target_type: Plate
+  default_state: passed
```

Comment:
don't think default state is passed on these XP plates. maybe for the cherrypick as its an input plate

Reply:
although I now see there are submissions for each of these plates so not sure

Reply:
Used here - https://github.com/sanger/sequencescape/blob/2f7cdbe7579a3075d6d1b878dd65d60186a0feff/app/models/transfer/state.rb#L30

So when trying to determine a plate's state, it looks first at the state of the transfer requests. If it can't determine the state from the transfer requests (there are 0 or >1 that aren't in the expected list), then it'll use the default state.

So I suppose for plates that are the first in a pipeline, they won't have transfer requests coming into them, so it'll use the default state. That's the cherrypick plate but not the other two.

Got a meeting so can't continue looking into it now, but guess it's also relevant to think about how the submission affects the state (can't remember where that code is).